test/evergreen.yml

```diff
@@ -3616,6 +3616,27 @@ tasks:
       - func: "upload-perf-test-stats"
         vars:
           perf-test-name: 500m-btree-populate.wtperf
+      - func: "upload artifact"
+        vars:
+          upload_filename: WT_TEST.tgz
+          upload_source_dir: wiredtiger/cmake_build/bench/wtperf/WT_TEST_0_0/
+      - func: "cleanup"
+
+  - name: perf-test-500m-btree-50r50u
+    tags: ["long-perf"]
+    depends_on:
+      - name: compile
+      - name: perf-test-500m-btree-populate
+    commands:
+      - command: timeout.update
+        params:
+          exec_timeout_secs: 21600
+          timeout_secs: 21600
+      - func: "fetch artifacts"
```

Comment:
Can you add a comment here (and in other tasks) explaining that the first `fetch artifacts` call gets the compiled binaries, and the second call pulls down only the WT_TEST folder? It's not obvious at first glance.

Reply:
Thanks